packages/sdk-v2/src/sdk/constants.ts

```diff
@@ -1,4 +1,16 @@
-import { EnvKey } from "./claim/sdk";
+import { EnvKey } from "./base/sdk";
+import { Token } from "@usedapp/core";
+import contractsAddresses from "@gooddollar/goodprotocol/releases/deployment.json";
+
+/* List of supported chains for this sdk. */
+export enum SupportedChainId {
+  FUSE = 122,
```

Comment:
why only fuse and celo? we have mainnet also

Reply:
ye, we only have currently celo/fuse functionalities
but we ofcourse will add stuff for mainnet in the future as well 
added now